kubeproxy/controllers/kubeproxy_controller.go

```diff
@@ -193,7 +193,7 @@ func OverrideApiserver(ctx context.Context, o declarative.DeclarativeObject, man
 			}
 			if master == "" {
 				master = "kubernetes-master"
-				klog.Warningf("using fallback for KUBERNETES_SERVICE_HOST: %v", master)
+				klog.InfoS("using fallback for KUBERNETES_SERVICE_HOST", "masterName", master)
```

Comment:
It change the log-level (from waning to info), but klog doesn't have WarningS, it's reasonable I think.
FYI: [Topics for adding WarningS](https://github.com/kubernetes/klog/pull/184) exists on k/klog.